### PR TITLE
draft branch for ssl:le:jobstatus command

### PR DESCRIPTION
**Pull request by dshelikhov**

### CODE REVIEW

**Comment by dshelikhov** [comment]
```
$ ./bin/atomic ssl:le:register 13597 admin.peterwpadmin.pagelyapps.com
Successfully queued job: 43960
Check status at: https://mgmt.pagely.com/api/ssl/letsencrypt/status/43960
```

```
./bin/atomic ssl:le:jobstatus 43960
In RequestException.php line 113:
  Client error: `GET https://mgmt.pagely.com/api/ssl/letsencrypt/status/43960` resulted in a `404 Not Found` response:  
  {"message":null}      
```

Need to speak to dev about how to do this, it probably requires changing auth scopes and that route might not even be available on atomic

mgmt api client makes when searching by fqdn:

```
$ mgmt ssl:le:jobstatus --accountId 13597 --fqdn admin.peterwpadmin.pagelyapps.com
...
...
+----------------+--------------------------------------------------------------------------+
| id             | 26487                                                                    |
| jobId          | 43962                                                                    |
| accountId      | 13597                                                                    |
| domain         | admin.peterwpadmin.pagelyapps.com                                        |
| step           | FETCH_CERTIFICATE                                                        |
| status         | COMPLETE                                                                 |
| message        | null                                                                     |
| certId         | 28804                                                                    |
| aliasId        | 142414                                                                   |
| appId          | 37397                                                                    |
| alternateNames | []                                                                       |
| created        | {"date":"2021-04-13 09:56:49.000000","timezone_type":3,"timezone":"UTC"} |
| modified       | {"date":"2021-04-13 09:56:56.000000","timezone_type":3,"timezone":"UTC"} |
+----------------+--------------------------------------------------------------------------+
````

**Comment by dshelikhov** [comment]
endpoint expects the DB record ID of the letsencrypt job as its input parameter (not the `jobId`), so it works: 

```
$ ./bin/atomic ssl:le:jobstatus 26487 | jq
{
  "@type": "Pagely.Model.SSL.LetsEncryptJob",
  "id": 26487,
  "jobId": 43962,
  "accountId": 13597,
  "domain": "admin.peterwpadmin.pagelyapps.com",
  "step": "FETCH_CERTIFICATE",
  "status": "COMPLETE",
  "message": null,
  "certId": 28804,
  "aliasId": 142414,
  "appId": 37397,
  "alternateNames": [],
  "created": "2021-04-13T09:56:49+00:00",
  "modified": "2021-04-13T09:56:56+00:00"
}

```

File: src/Command/SSL/LetsencryptJobStatusCommand.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+class LetsencryptJobStatusCommand extends AbstractSSLCmd
+{
+    protected $commandName = 'ssl:le:jobstatus';
+
+    public function configure()
+    {
+        parent::configure();
+        $this->setDescription('Check letsencrypt job status')
+            ->addArgument('id', InputArgument::REQUIRED, 'ID of the letsencrypt job (not jobId)')
```
Comment (armanly) [inline src/Command/SSL/LetsencryptJobStatusCommand.php:17]: is the text `(not jobId)` supposed to say `(not appid)` ?

Comment (baohx2000) [reply src/Command/SSL/LetsencryptJobStatusCommand.php:17]: It is valid... there is a Pagely job queue id (jobId) and a letsencrypt job id.

File: src/API/SSL/SSLClient.php [hunk 2]
```diff
@@ -12,6 +12,11 @@ public function getCert($accessToken, $id, $includeChain = false)
         return $this->guzzle($this->getBearerTokenMiddleware($accessToken))->get("ssl/certs/{$id}", ['query' => ['includeChain' => $includeChain]]);
     }
 
+    public function getLetsencryptStatus($accessToken,  $id)
+    {
+        return $this->guzzle($this->getBearerTokenMiddleware($accessToken))->get("/api/ssl/letsencrypt/status/{$id}");
```
Comment (baohx2000) [inline src/API/SSL/SSLClient.php:17]: Fix path for consistency
```suggestion
        return $this->guzzle($this->getBearerTokenMiddleware($accessToken))->get("ssl/letsencrypt/status/{$id}");
```

**Pull request merged**